src/ontology/edits/oeo-physical.omn

```diff
@@ -793,2170 +835,2451 @@ Generally, flows in the model can not be controlled but are a function of the re
+        OEO_00000334
     
     
-Class: Fusion
+Class: OEO_00000178
 
     Annotations: 
         <http://purl.obolibrary.org/obo/IAO_0000115> "In nuclear chemistry, nuclear fusion is a reaction in which two or more atomic nuclei are combined to form one or more different atomic nuclei and subatomic particles (neutrons or protons). The difference in mass between the reactants and products is manifested as either the release or absorption of energy."@en,
-        <http://purl.obolibrary.org/obo/IAO_0000119> "https://en.wikipedia.org/w/index.php?title=Nuclear_fusion&oldid=906947069"@en
+        <http://purl.obolibrary.org/obo/IAO_0000119> "https://en.wikipedia.org/w/index.php?title=Nuclear_fusion&oldid=906947069"@en,
+        rdfs:label "fusion"
     
     SubClassOf: 
-        NuclearEnergyProduction
+        OEO_00000301
     
     
-Class: GasDieselOil
+Class: OEO_00000181
 
     Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "Gas/diesel oil is an OilAndPetroleumProduct that is primarily a medium distillate distilling between 180 °C and 380 °C. Includes blending components. Several grades are available depending on uses."@en,
-        <http://purl.obolibrary.org/obo/IAO_0000119> "REGULATION (EC) No 1099/2008 OF THE EUROPEAN PARLIAMENT AND OF THE COUNCIL of 22 October 2008 on energy statistics https://eur-lex.europa.eu/legal-content/EN/TXT/HTML/?uri=CELEX:32008R1099&from=EN"@en
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Gas/diesel oil is an oil-and-petroleum-product that is primarily a medium distillate distilling between 180 °C and 380 °C. Includes blending components. Several grades are available depending on uses."@en,
+        <http://purl.obolibrary.org/obo/IAO_0000119> "REGULATION (EC) No 1099/2008 OF THE EUROPEAN PARLIAMENT AND OF THE COUNCIL of 22 October 2008 on energy statistics https://eur-lex.europa.eu/legal-content/EN/TXT/HTML/?uri=CELEX:32008R1099&from=EN"@en,
+        rdfs:label "gas diesel oil"
     
     SubClassOf: 
-        OilAndPetroleumProducts,
-        has_normal_state_of_matter value liquid
+        OEO_00000309,
+        has_normal_state_of_matter value OEO_00000256
     
     
-Class: GasPowerplant
+Class: OEO_00000183
 
     Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "A GasPowerplant is a Powerplant having an aggregate of GasFiredPowerUnits as its PowerGeneratingUnits."@en
+        <http://purl.obolibrary.org/obo/IAO_0000115> "gasoline (American English) or petrol (British English) is an OilAndPetroleumProduct in the form of a transparent petroleum-derived liquid that is used primarily as a fuel in spark-ignited internal combustion engines. It consists mostly of organic compounds obtained by the fractional distillation of petroleum, enhanced with a variety of additives.",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "https://en.wikipedia.org/w/index.php?title=Gasoline&oldid=867948640",
+        rdfs:label "gasoline"
```

Comment:
If we have petrol as an alternative term to gasoline shouldn't we the include petrol as a second label?

Either fix this quickly here in this PR or create an issue if this needs discussion.

Reply:
I added it as an "alternative term"